src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/VariantsToTable.java

```diff
@@ -91,30 +109,36 @@
 
     @Argument(fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME,
             shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME,
-            doc="File to which results should be written (defaults to stdout)")
+            doc="File to which the tab-delimited table is written (defaults to stdout)")
     private String out = null;
 
     /**
      * -F NAME can be any standard VCF column (CHROM, ID, QUAL) or any annotation name in the INFO field (e.g., -F AC).
      * To capture GENOTYPE (FORMAT) field values, see the -GF argument. This argument accepts any number
      * of inputs.  So -F CHROM -F POS is allowed.
      */
-    @Argument(fullName="fields", shortName="F", doc="The name of each field to capture for output in the table", optional=true)
+    @Argument(fullName="fields",
+            shortName="F",
+            doc="The name an INFO field to include in the output table", optional=true)
     private List<String> fieldsToTake = new ArrayList<>();
 
     /**
      * -GF NAME can be any annotation name in the FORMAT field (e.g., GQ, PL).
      * This argument accepts any number of inputs.  So -GF GQ -GF PL is allowed.
      */
-    @Argument(fullName="genotypeFields", shortName="GF", doc="The name of each genotype field to capture for output in the table", optional=true)
+    @Argument(fullName="genotype-fields",
+            shortName="GF",
+            doc="The name a genotype field to include in the output table", optional=true)
     private List<String> genotypeFieldsToTake = new ArrayList<>();
 
     /**
      * By default this tool only emits values for records where the FILTER field is either PASS or . (unfiltered).
      * Using this flag will cause VariantsToTable to emit values regardless of the FILTER field value.
      */
     @Advanced
-    @Argument(fullName="showFiltered", shortName="raw", doc="If provided, field values from filtered records will be included in the output", optional=true)
+    @Argument(fullName="show-filtered",
+            shortName="raw",
+            doc="Field values from filtered records will be included in the output", optional=true)
```

Comment:
doc="Include fields from filtered records in the output"

Reply:
done